pkg/kubelet/dockertools/docker_cache.go

```diff
@@ -23,6 +23,7 @@ import (
 
 type DockerCache interface {
 	RunningContainers() (DockerContainers, error)
+	ForceUpdateIfOlder(time.Time) error
 }
 
 func NewDockerCache(client DockerInterface) (DockerCache, error) {
```

Comment:
One thing I'd recommend is this in the cache and the cache_test:

```Go
var _ DockerCache = new(dockerCache)
```

this ensures that dockerCache abides by the DockerCache interface. If not, you'll get a compile error. Not needed for this PR though.